src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -49,6 +52,8 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
         super.generateColumns();
 
         // override default visibility for some columns
+        this._columns[MutationTableColumnType.TUMOR_ALLELE_FREQ].visible = TumorAlleleFreqColumnFormatter.isVisible(
+            (this.props.dataStore as IMobXApplicationDataStore<Mutation[]>).allData as Mutation[][]);
```

Comment:
I don't think you need to do casting in this class. Can you double check? And if that's the case remove the casting and the corresponding import statement?